.github/workflows/deploy-apis-to-production.yml

```
@@ -1732,3 +1744,62 @@ jobs:
+          credentials_json: ${{ secrets.GCP_SA_CREDENTIALS }}
+
+      - name: Setup Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1.1.1
+
+      - name: Update the corresponding k8s configmap(s)
+        run: |
+          cd src/website/
+          gcloud secrets versions access latest --secret="prod-env-website-backend" > .env
+          kubectl create configmap --dry-run=client -o yaml --from-env-file=.env prod-website-api-config | kubectl replace -f - -n production
+          gcloud secrets versions access latest --secret="prod-key-device-monitoring-service-account" > google_application_credentials.json
+          kubectl create configmap --dry-run=client -o yaml --from-file=google_application_credentials.json prod-website-api-config-files | kubectl replace -f - -n production
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)